automated/linux/ltp/skipfile-lkft.yaml

```diff
@@ -24,7 +24,7 @@ skiplist:
     url: https://bugs.linaro.org/show_bug.cgi?id=3719
     environments: all
     boards:
-      - all 
+      - all
```

Comment:
we should not have to add  this to the PR.

Reply:
This should be a check that should be added to the upstream repository and fixed there before we go live with your PRs.